app/src/test/java/android/util/Log.java

```diff
@@ -0,0 +1,35 @@
+package android.util;
+
+/**
+ * Workaround slf4j trying to invoke Android.Log.isLoggable
+ * by providing a shim which simply uses println on everything.
+ */
+public class Log {
```

Comment:
If it's slf4j causing this issue we should probably drop slf4j-android from the test dependencies and use another logger. No need to add this class then.

Reply:
We could use the [impl](https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-slf4j-impl) for tests only?

For our Gradle files, how can we structure it so we only include `slf4j-android` in our app and not tests?

Reply:
> We could use the impl for tests only?

I'm not sure why you're linking to a log4j module, we don't use log4j.

> For our Gradle files, how can we structure it so we only include slf4j-android in our app and not tests?

I looked a bit into this and it doesn't seem that easy



I think a good solution here is to replace the slf4j-android dependency with something else, I'm looking into that and will open a pull request for that. So we can keep this class for now.